sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -14,13 +14,33 @@
       <h2 class="category-header-large primary-color pb-4">
         Apply for funding
       </h2>
-      <content-group>
-        <p class="category-header-medium-small pa-2">
+      <content-group
+        ><p class="category-header-medium-small pa-2">
           Financial Aid Application
         </p>
+        <p class="px-2">
+          In what capacity will you be attending this program?
+          <tooltip-icon
+            >A student is considered to be in part-time studies when taking
+            between 20% and 59% of a full-time course load in a course or
+            continuous period of study.</tooltip-icon
+          >
+          <span class="field-required"></span>
```

Comment:
Is there an intention to reuse the form.io styles to display the same "*" for a Vuetify component?
Was it part of a requirement? Even if it is, it demands more from accessibility besides only adding the red asterisk to the top right.
If it was not part of the requirement, please remove it.